code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -195,5 +195,6 @@
 				/obj/item/weapon/spacecash/c200 = 1,
 				/obj/item/weapon/airlock_electronics = 1,
 				/obj/item/weapon/gun/energy/kinetic_accelerator/super = 1,
-				/obj/item/pizzabox = 3,
+				/obj/item/pizzabox/margherita = 1,
+				/obj/item/pizzabox/meat = 1,
```

Comment:
Really don't think these should be added to the maintenance loot table in this PR.